lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs

```diff
@@ -548,15 +566,139 @@ testRule({
 			endColumn: 25,
 		},
 		{
-			code: 'a { padding: calc(1rem +  \t\r\n  1em)}',
-			fixed: 'a { padding: calc(1rem +\r\n  1em)}',
+			code: 'a { padding: calc(1rem +  \t\r\n  1em); }',
+			fixed: 'a { padding: calc(1rem +\r\n  1em); }',
 			description: 'several whitespace characters after operator but before the \\r\\n',
 			message: messages.expectedAfter('+'),
 			line: 1,
 			column: 24,
 			endLine: 1,
 			endColumn: 25,
 		},
+		{
+			code: 'a { padding: calc(1px +-2px); }',
+			fixed: 'a { padding: calc(1px + -2px); }',
+			message: messages.expectedAfter('+'),
+			line: 1,
+			column: 23,
+			endLine: 1,
+			endColumn: 24,
+		},
+		{
+			code: 'a { padding: calc(1px+2px+3px); }',
```

Comment:
This one should be `'a { padding: calc(1px+2px + 3px); }'` to catch regressions.
cf #7181

Reply:
But, autofixing to `calc(1px + 2px + 3px)` is better than `calc(1px+2px + 3px)`, isn't it?

Reply:
Look at the [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0BDABMAOgOzWg9AFRoCiATmRGWofjngC4QAO8aYKANmABQCszAB4BqAMxCAlAG4cAXxw4kAIwz0CxAHIQ0MClRp1caJq3ZdeAkeMFoAtGgBMkmdnnYQAGhAAzAJacYDRQAWzhEGEEQ5gCkMABnOM9wCGw-AHMEECwjTBAyAFcAuNy2bLw8XO987DAGXxTbDm5bbAhbarjmFDAYKFsWXRQmMhLjApg1NzckyFTfNIAxKmChzIArOJSk2GZExDK0XLiGAE8Azl9sBlHcziGYY9yPNSPT88uGRpT022OUbCgKDIUBuIDuDAe1xAchAsiAA) linked in the issue; as you can see it's not detected as invalid right now but `calc(1px + 2px + 3px)` is.
Hence we want `calc(1px+2px + 3px)` way more because it is/was a bug.
i.e. we want to pick an eventual regression